### PR TITLE
Add Prettier support

### PR DESCRIPTION
**Pull request by lipis**
#727 #725 
https://prettier.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,22 @@
-gae-init
-========
+# gae-init
 
 [![Slack Status](https://gae-init-slack.herokuapp.com/badge.svg)](https://gae-init-slack.herokuapp.com)
 
-> **gae-init** is the easiest boilerplate to kick start new applications on Google
-App Engine using Python, Flask, RESTful, Bootstrap and tons of other cool features.
+> **gae-init** is the easiest boilerplate to sdf start new applications sdf sd sd on Google App sdf using Python, Flask, RESTful, sdfele and tons of sdf other cool sdf.
```
Comment (rbubley) [inline README.md]: Fat finger?

Comment (lipis) [reply README.md]: hehe.. thanks :)

**Comment by lipis** [comment]
Anything else?

**Pull request merged**